src/zc/lockfile/tests.py

```diff
@@ -102,4 +194,9 @@ def test_suite():
     suite.addTest(doctest.DocTestSuite(
         setUp=setupstack.setUpDirectory, tearDown=setupstack.tearDown,
         checker=checker))
+    # Add doctests from zc/lockfile/__init__.py
+    suite.addTest(doctest.DocTestSuite(zc.lockfile))
```

Comment:
This is unrelated to the rest of the changes, yes?  Because I don't see you adding any new doctests to `__init__.py`.  Are there any old ones that were mistakenly exclude?